public/app/plugins/datasource/loki/components/LokiLabelBrowser.tsx

```diff
@@ -138,6 +148,8 @@ const getStyles = (theme: GrafanaTheme2) => ({
     text-overflow: ellipsis;
     transition: opacity 100ms linear;
     opacity: 0;
+    font-size: ${theme.typography.pxToRem(12)};
```

Comment:
What if we used `${theme.typography.bodySmall.fontSize}`? We already use it in Loki editor. So it could maybe be more consistent.

Reply:
Good catch! Implemented